axelrod/player.py

```diff
@@ -122,8 +140,15 @@ def set_match_attributes(self, length=-1, game=None, noise=0):
         self.receive_match_attributes()
 
     def __repr__(self):
-        """The string method for the strategy."""
-        return self.name
+        """The string method for the strategy.
+        Appends the `__init__` parameters to the strategy's name."""
+        name = self.name
+        prefix = ': '
+        gen = (value for value in self.init_kwargs.values() if value is not None)
+        for value in gen:
+            name = ''.join([name, prefix, str(value if not isinstance(value, float) else round(value, 2))])
```

Comment:
Originally, the `__repr__` of Random was doing a round. When I print all strategies `__repr__` with default values I get srings a bit long for those :
```
ZD-Extort-2: 0.1111111111111111, 0.5
ZD-Extort-2 v2: 0.125, 0.5, 1
ZD-Extort-4: 0.23529411764705882, 0.25, 1
```
So maybe use a bigger value for round or, if you prefer to be really precise and you don't mind `__repr__` like my example above, I can suppress it